LICENSE.md

```diff
@@ -0,0 +1,8 @@
+# Copyright holders
+
+2021-2022 Alexander Müller
+2022      Tarun Kumar Mitruka Vinod Kumar Mitruka
```

Comment:
Thanks !

Reply:
Do you want here a different name or is this ok like that?

Reply:
This is perfect 👍

Reply:
done